examples/batched-processing/README.md

```diff
@@ -0,0 +1,34 @@
+# Batched Processing
+
+This example is an example of a minimal two-server aggregation scheme that
+fulfils the privacy guarantees of the Prio system.
+
+## Building
+
+docker build -t prio-dev ../..
+
+
```

Comment:
Probably want a section on running as well?

Reply:
Moved this into a quickstart section